docs/cloud-native-security/wiz.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[ingest-wiz-data]]
+= Ingest Wiz data
+
+In order to enrich your {elastic-sec} workflows with third-party cloud security posture and vulnerability data collected by Wiz:
+
+* Follow the steps to {integrations-docs}/wiz[set up the Wiz integration]. 
+
+* Make sure the integration version is at least 2.0.1. 
+
+* Ensure you have `read` privileges for the following indices: `security_solution-*.misconfiguration_latest`, `security_solution-*.vulnerability_latest`.
+
+* While configuring the Wiz integration, turn on **Cloud Configuration Finding logs** and **Vulnerability logs**. We recommend you also set the **Initial Interval** values for both settings to `2160h` (equivalent to 90 days) to ingest existing logs.
+
+image::images/wiz-config-finding-logs.png[Wiz integration settings showing the findings toggle]
+
+image::images/wiz-config-vuln-logs.png[Wiz integration settings showing the vulnerabilities toggle]
+
+After you've completed these steps, Wiz data will appear on the <<cspm-findings-page, Misconfiguations>> and <<vuln-management-findings, Vulnerabilities>> tabs of the Findings page . 
```

Comment:
Not even sure if this extra space would render in output, but just in case:
```suggestion
After you've completed these steps, Wiz data will appear on the <<cspm-findings-page, Misconfiguations>> and <<vuln-management-findings, Vulnerabilities>> tabs of the Findings page.
```